challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.collect(&:capitalize)
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.find_all { |animals| animals == "dog"  }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { |animals| animals == "dog"  }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count 
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.first(2)
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.find_all { |string| string[0] == "c" || string[0] == "d"}
```

Comment:
This can be simplified a bit with [`String#start_with?`](http://ruby-doc.org/core-2.2.2/String.html#method-i-start_with-3F).